hoprd/db/api/src/aliases.rs

```diff
@@ -40,53 +44,75 @@ impl HoprdDbMetadataOperations for HoprdDb {
     async fn get_aliases(&self) -> Result<Vec<Alias>> {
         let rows = hoprd_db_entity::aliases::Entity::find().all(&self.metadata).await?;
 
-        let aliases = rows
-            .iter()
-            .map(|row| Alias {
-                peer_id: row.peer_id.clone(),
-                alias: row.alias.clone(),
-            })
-            .collect();
+        let aliases: Vec<Alias> = rows.iter().map(|row| Alias::from(row.to_owned())).collect();
```

Comment:
If you do `into_iter()` you can get rid of the `.to_owned()` conversion and it will simplify the line to:
`rows.into_iter().map(Alias::from).collect()`

Prefer `.into_iter()` over `.iter()` anywhere, where you do not need the input vector any further.